README.md

```diff
@@ -68,6 +68,9 @@ Other supported formats are listed below.
 * `multisubnetfailover`
   * `true` (Default) Client attempt to connect to all IPs simultaneously. 
   * `false` Client attempts to connect to IPs in serial.
+* `sendStringParametersAsUnicode`
+  * `true` (Default) Go default string types sent as `nvarchar`.
+  * `false` Go default string types sent as `varchar`.
```

Comment:
>ing types sent as `varchar`. [](http://example.com/codeflow?start=26&length=28)

without any way of indicating the code page, how does this work? Go strings are UTF8

Reply:
@shueybubbles it's upto the clients to decide how they want to use Go strings. In the same way other drives like `postgres` for Go also works.

Reply:
I'm not convinced it's a good idea for the driver to have such a broad toggle without a more comprehensive fix for mixing string encodings AND having a way for clients to specify the code page of the input.

Note the driver already has a way to designate input parameter strings as varchar to avoid the conversion - you need to use the driver-specific [VarChar](https://github.com/microsoft/go-mssqldb/blob/ada30cbaf031d81144e5c5487ddefd4f3dac0ef5/mssql_go19.go#L33) 
Using that type has the benefit of apps being explicit about the desired conversion behavior and being able to mix unicode strings with non-unicode strings in the same connection.

Reply:
@shueybubbles can you help close this? This has been pending since long. We can have a quick connect on this if needed.